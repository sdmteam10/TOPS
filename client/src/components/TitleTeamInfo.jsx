import React, { Component } from 'react';

class TitleTeamInfo extends Component {
    state = {

    }

    render() {
        return (
          <div className="bg-primary font-monospace" style={{ height: "auto" }}>
            <div className='mx-auto text-center'>
            <br />
              <h2 className="text-light">Team Information</h2>
              <br />
              <h6 className="text-light"> Fill in the form quickly.</h6>
              <h6 className="text-light"> Record the interview. </h6>
              <h6 className="text-light"> Automatic generate activity suggestion. </h6>
              <br />
            </div>
          </div>
        )
    }
}

export default TitleTeamInfo