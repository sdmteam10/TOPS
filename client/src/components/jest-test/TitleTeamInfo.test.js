// TitleTeamInfo.test.js
import React from 'react'
import TitleTeamInfo from '../TitleTeamInfo'
import renderer from 'react-test-renderer';
import { fireEvent, render } from "@testing-library/react"

//it.only() "only test this <it>."
//it.skip() "not test this <it>."
// done() under expect(){} -  should be only used for async functions

  describe('Test <TitleTeamInfo /> component', function() {
    it('Should render', function() {  
      //Render component should be successed.
      const component = renderer.create(
        <TitleTeamInfo />
      );
        let testSnap = component.toJSON();
        expect(testSnap).toMatchSnapshot(); 
    })
  })