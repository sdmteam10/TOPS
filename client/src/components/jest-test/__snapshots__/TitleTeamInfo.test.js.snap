// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <TitleTeamInfo /> component Should render 1`] = `
<div
  className="bg-primary font-monospace"
  style={
    Object {
      "height": "auto",
    }
  }
>
  <div
    className="mx-auto text-center"
  >
    <br />
    <h2
      className="text-light"
    >
      Team Information
    </h2>
    <br />
    <h6
      className="text-light"
    >
       Fill in the form quickly.
    </h6>
    <h6
      className="text-light"
    >
       Record the interview. 
    </h6>
    <h6
      className="text-light"
    >
       Automatic generate activity suggestion. 
    </h6>
    <br />
  </div>
</div>
`;
